### ISSUE TITLE
Exclude list-structures, List, Seq and Array from rule

### ISSUE THREAD

**Issue by kgrunert** [jatcwang/scalafix-named-params#70]
In my `.scalafix.conf` I have set:

```
UseNamedParameters {
    minParams = 2
}

rules = [
    OrganizeImports,
    UseNamedParameters,
    RemoveUnused,
    RedundantSyntax,
    NoValInForComprehension,
    ProcedureSyntax,
    LeakingImplicitClassVal
]
```

and yet, when i run `scalafixAll` it adds a named parameters to scala-data structures such as `Seq` and `Array`:

```
// before:
      Array(
        messageWithX,
        messageWithY,
        messageWithZ
      )
```

```
// after: 
      Array(
        xs = messageWithX,
        messageWithY,
        messageWithZ
      )
```

and I understand, it is correct behaviour, since I have configured `minParams = 2`, but wouldnt it be nicer to exclude these structures from the rule (or at least make it configurable?)
What do you think?

**Comment by jatcwang** [comment]
Perhaps I should make it so we count number of non-varargs parameters in the called method, which seems more general and shouldn't require minimal user configuration.

i.e. these should all work OOTB
```
// minParams = 3
// ok
Array(
  1,
  2,
  3
)

// ok
TwoParamLastOneIsVarArg(
  "first",
  1,
  2,
  3
)

// will be fixed
ThreeParamLastOneIsVarArg(
  "first",
  "second",
  1,
)
```

**Comment by kgrunert** [comment]
That sounds good! So that varargs-count is not being considered in the minParams evaluation

**Comment by jatcwang** [comment]
Fixed in 0.2.1
